src/common/BrokerDetailsBreadcrumb.tsx

```diff
@@ -0,0 +1,27 @@
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import { useTranslation } from '../i18n';
+
+const BrokerContainerBreadcrumb = (props: { currentName: string }) => {
```

Comment:
Create type for props instead.
export type BrokerDetailsBreadcrumbProps:{
  name: string;
};

```suggestion
const BrokerDetailsBreadcrumb: FC<BrokerDetailsBreadcrumbProps> = ({name}) => {
```